contrib/supervisord.conf

```diff
@@ -17,26 +17,32 @@ password = 🍟
 [rpcinterface:supervisor]
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
-[program:redis]
```

Comment:
Any reason redis was dropped from supervisord?

Reply:
This feels like it should be running at the system level. Postgres, ES, etc are all started at the system level because system setup varies for each. For instance, I run all of these in docker containers.

Does the redis target here feel important? We can keep it if so.

Reply:
Not really. I actually don't use supervisord, I run each program in separated shells.

Reply:
>  I run each program in separated shells.

Me too for now. Although, I think we should settle on one way to run our development instance.

I'm not sold on supervisor, but it looks the best for now IMO.